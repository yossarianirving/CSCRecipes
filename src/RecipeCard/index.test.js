import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { RecipeCard } from '.';

describe('RecipeCard', () => {
  const props = {
    title: "Beef stew",
    field_images: "/images/test/blah.jpeg",
    field_summary: "Easy to make beef stew"
  }
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<RecipeCard { ...props } />, div)
    ReactDOM.unmountComponentAtNode(div);
  })

  test('has a valid snapshot', () => {
    const component = renderer.create(
    <RecipeCard { ...props } />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    });
})