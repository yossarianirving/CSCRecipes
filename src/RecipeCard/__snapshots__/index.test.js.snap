// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeCard has a valid snapshot 1`] = `
<div
  className="recipe-card"
>
  <img
    alt="Beef stew"
    className="recipe-card-img"
    src="http://gtest.dev.wwbtc.com/images/test/blah.jpeg"
  />
  <button
    className="recipe-card-title"
    onClick={[Function]}
  >
    Beef stew
  </button>
  <div
    className="recipe-card-summary"
  >
    Easy to make beef stew
  </div>
</div>
`;
